testimonials.html

```diff
@@ -0,0 +1,62 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <link rel="stylesheet" href="./assets/css/testimonials.css">
+</head>
+<body>
+    <div class="testimonial" id="testimonials">
```

Comment:
Add Navigation Menu similar to `index.html`.
You can just copy-paste the nav bar html and link index.css to testimonial.html